From da9fb98442e7960a30ae6f2183c9c1fd89a0855b Mon Sep 17 00:00:00 2001
From: Jean-Philippe Lang <jp_lang@yahoo.fr>
Date: Sun, 25 Jun 2017 15:15:23 +0000
Subject: [PATCH 01/11] Fix HTML structure of query_form on calendar/gantt
 views (#26125).

This unifies the HTML structure of the query form on the calendars/show
and gantts/show views to be the same as it is currently for the issues
and spent time views, which were themself unified into the new
queries/_query_form partial in r15649.

Patch by Mischa The Evil.

git-svn-id: http://svn.redmine.org/redmine/trunk@16701 e93f8b46-1217-0410-a6f0-8f06a7374b81
---
 app/views/calendars/show.html.erb | 5 +++++
 app/views/gantts/show.html.erb    | 5 +++++
 2 files changed, 10 insertions(+)

diff --git a/app/views/calendars/show.html.erb b/app/views/calendars/show.html.erb
index f18946027..45de7a6cf 100644
--- a/app/views/calendars/show.html.erb
+++ b/app/views/calendars/show.html.erb
@@ -3,6 +3,9 @@
 <%= form_tag({:controller => 'calendars', :action => 'show', :project_id => @project},
              :method => :get, :id => 'query_form') do %>
 <%= hidden_field_tag 'set_filter', '1' %>
+
+<div id="query_form_with_buttons" class="hide-when-print">
+<div id="query_form_content">
 <fieldset id="filters" class="collapsible <%= @query.new_record? ? "" : "collapsed" %>">
   <legend onclick="toggleFieldset(this);"><%= l(:label_filter_plural) %></legend>
   <div style="<%= @query.new_record? ? "" : "display: none;" %>">
@@ -13,6 +16,7 @@
 <p style="float:right;">
   <%= link_to_previous_month(@year, @month, :accesskey => accesskey(:previous)) %> | <%= link_to_next_month(@year, @month, :accesskey => accesskey(:next)) %>
 </p>
+</div>
 
 <p class="buttons">
 <%= label_tag('month', l(:label_month)) %>
@@ -23,6 +27,7 @@
 <%= link_to_function l(:button_apply), '$("#query_form").submit()', :class => 'icon icon-checked' %>
 <%= link_to l(:button_clear), { :project_id => @project, :set_filter => 1 }, :class => 'icon icon-reload' %>
 </p>
+</div>
 <% end %>
 
 <%= error_messages_for 'query' %>
diff --git a/app/views/gantts/show.html.erb b/app/views/gantts/show.html.erb
index 314c2f2c5..242489ff0 100644
--- a/app/views/gantts/show.html.erb
+++ b/app/views/gantts/show.html.erb
@@ -14,6 +14,9 @@
              :method => :get, :id => 'query_form') do %>
 <%= hidden_field_tag 'set_filter', '1' %>
 <%= hidden_field_tag 'gantt', '1' %>
+
+<div id="query_form_with_buttons" class="hide-when-print">
+<div id="query_form_content">
 <fieldset id="filters" class="collapsible <%= @query.new_record? ? "" : "collapsed" %>">
   <legend onclick="toggleFieldset(this);"><%= l(:label_filter_plural) %></legend>
   <div style="<%= @query.new_record? ? "" : "display: none;" %>">
@@ -53,6 +56,7 @@
     </table>
   </div>
 </fieldset>
+</div>
 
 <p class="contextual">
   <%= gantt_zoom_link(@gantt, :in) %>
@@ -76,6 +80,7 @@
                        :class => 'icon icon-save' %>
 <% end %>
 </p>
+</div>
 <% end %>
 
 <%= error_messages_for 'query' %>
-- 
2.19.0.windows.1

